config/manifests/kustomization.yaml

```diff
@@ -3,7 +3,6 @@
 resources:
 - bases/build-service.clusterserviceversion.yaml
```

Comment:
this file is also not there, in general I thing that whole config/manifests can be removed since it's not used at all

Reply:
Done.